src/Compilers/CSharp/Portable/Symbols/SubstitutedNamedTypeSymbol.cs

```diff
@@ -159,6 +159,9 @@ internal override ImmutableArray<NamedTypeSymbol> GetInterfacesToEmit()
             throw ExceptionUtilities.Unreachable;
         }
 
+        internal override bool GetUnificationUseSiteDiagnosticRecursive(ref DiagnosticInfo result, Symbol owner, ref HashSet<TypeSymbol> checkedTypes)
+            => OriginalDefinition.GetUnificationUseSiteDiagnosticRecursive(ref result, owner, ref checkedTypes);
```

Comment:
>OriginalDefinition [](http://example.com/codeflow?start=15&length=18)

Do we need to check `TypeArguments` as well?

Reply:
The TypeArguments are just the alpha-renamed type parameters from the `OriginalDefinition`, there's nothing to check.